ee-9/source-transform/microprofile/pom.xml

```diff
@@ -45,6 +45,7 @@
         <module>fault-tolerance-smallrye</module>
         <module>openapi-smallrye</module>
         <module>reactive-streams-operators-smallrye/cdi-provider</module>
+        <module>health-smallrye</module>
```

Comment:
Just a nitpick but this one is out of order, the general idea is to keep items in poms sorted alphabetically.